client/tokenized-express-checkout/__tests__/tokenized-express-checkout--product-page.test.js

```diff
@@ -148,7 +164,7 @@ describe( 'Tokenized Express Checkout Element - Product page logic', () => {
 		} );
 
 		// triggering the `ready` event on the ECE button, to test its callback.
-		stripeElementMock[ `__triggerEvent-ready` ]( {
```

Comment:
I slightly changed the implementation I made on [this PR](https://github.com/Automattic/woocommerce-payments/pull/10592) I just merged to (hopefully) make retrieving the registered event a little clearer, without the template function name 🤷